python-backend/app/api/mines/status/models/mine_status.py

```diff
@@ -55,18 +49,27 @@ def create_mine_status_labels_list(self):
                 self.mine_status_xref.mine_operation_status_sub_reason.description)
         return status_labels_list
 
-    def json(self, show_mgr=True):
-        status_values_list = self.create_mine_status_values_list()
-        status_labels_list = self.create_mine_status_labels_list()
-        return {
-            'mine_status_guid': str(self.mine_status_guid),
-            'mine_guid': str(self.mine_guid),
-            'mine_status_xref_guid': str(self.mine_status_xref.mine_status_xref_guid),
-            'status_values': status_values_list,
-            'status_labels': status_labels_list,
-            'effective_date': self.effective_date.isoformat(),
-            'expiry_date': self.expiry_date.isoformat()
-        }
+    def __repr__(self):
+        return '<MineStatus %r>' % self.mine_status_guid
+
+    def validate_status_code_exists(self, mine_status_xref, mine_status_code, code_or_description):
+        try:
+            return mine_status_xref[mine_status_code][code_or_description]
+        except KeyError:
+            return None
+
+    # def json(self, show_mgr=True):
+    #     status_values_list = self.create_mine_status_values_list()
+    #     status_labels_list = self.create_mine_status_labels_list()
+    #     return {
+    #         'mine_status_guid': str(self.mine_status_guid),
+    #         'mine_guid': str(self.mine_guid),
+    #         'mine_status_xref_guid': str(self.mine_status_xref.mine_status_xref_guid),
+    #         'status_values': status_values_list,
+    #         'status_labels': status_labels_list,
+    #         'effective_date': self.effective_date.isoformat(),
+    #         'expiry_date': self.expiry_date.isoformat()
+    #     }
```

Comment:
Comment to remove? Why would the `mine_status` json function be changed/removed here?

Reply:
If its no longer used by any returns. I am not sure about that though so I have it commented out for now and would like some more rigorous testing.

Reply:
If the intention is to remove it (assuming that nothing blows up in manual testing) then we can just remove it. We have a git history of it if we ever need to get it back